_posts/2017/2017-02-28-Using common development environmet.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "One environment to rule them all"
+date: "2017-02-22 07:00:00"
+image: blog-header/ring.jpg
+categories: tech-stories
+authors: ["Jordi Viladoms"]
+---
+
+One of the most popular problems that developers have to face is to work with a common development environment. At the same time, this environment has to be the same that the production one. ePages software is working into servers with centOS or debian and it makes mandatory for the developers to write the code under this environment. For sure make no sense to force all the developers to have one of those OS installed at the laptop taking into account that the most preferred OS are [macOS, ubuntu or windows](https://developer.epages.com/blog/2016/12/22/the-battle-of-the-operating-systems.html).
+
+This scenario force us to find a solution. There are several ways to fix this problem and I'll try to explain the different options that we have and which one has been mainly chosen by Team Ocean Barcelona.
+
+## Convert your laptop into a VM container
+
+One of the easiest way to do it and still the most used for some developers is to use a VM like vmware or virtual box and add there the maximum resources of your laptop. Then this VM becomes the main OS of your laptop and all the tools (IDE, scripts, ...) are hosted inside. It means that in fact the laptop is hosting two operatives systems but mainly only one is used.
```

Comment:
One of the easiest ways to find a solution - and also the most popular one - is to use a VM like [vmware](http://www.vmware.com/) or [VirtualBox](https://www.virtualbox.org/) and add the maximum resources of your laptop to it.
Then, this VM becomes ...
It means, that in fact the laptop is hosting two operative systems but mainly only one is used.